storybook/src/elements/nuxeo-date/nuxeo-date.stories.js

```diff
@@ -0,0 +1,13 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date';
+import { select, date } from '@storybook/addon-knobs';
+
+storiesOf('UI/nuxeo-date', module).add('Nuxeo Date', () => {
+  const dateTime = date('Date', new Date('Mars 4, 2020'), '');
```

Comment:
Why use "Mars 4, 2020" and not the current date? Also, while this worked fine on Chrome / Safari, it broke down on Firefox, stating that the date is invalid. Can you reproduce this? It would appear that it does not follow a valid format (`Date.parse('Mars 4, 2020')` returns `NaN` on Firefox). I expect that relying on the current date will fix the issue:
```suggestion
  const dateTime = date('Date', new Date(), '');
```

Reply:
Indeed!

Reply:
Regarding @RSalem07 question about conventions.
In this specific case, those variables are knobs specific to that story and, for that reason, they should be inside 🙂 
More, they are not `CONST` like in Java, they are mapped to Storybook knobs and will take the value selected by the user on a dropdown, for instance.

Reply:
@Gabez0r thanks for your feedback! On the documentation it says specifically: "Note: the default value must not change - e.g., do not do date('Label', new Date()) or date('Label')." I can look into the default date and why it's not working on firefox.

Reply:
Well noted. 👍 Let's try have a look on Firefox, then.

Reply:
Did you understand why we shouldn't use `date('Label', new Date())`? I had a go at it and it worked, with no error.

As a matter of fact, the default value is actually `new Date()` (see https://github.com/storybookjs/storybook/blob/master/addons/knobs/src/index.ts#L100), so I guess we should be ok using it.

Reply:
@Gabez0r made the pr changes.

Reply:
Do we still want to have an hardcoded date? https://github.com/nuxeo/nuxeo-elements/pull/212/files#diff-1aa0ccfce68c5d10707d49d568810dadR7
I would be in favor of `new Date()` since it seems to be a misalignment on documentation and on what code does 😅